### ISSUE TITLE
Some error messages are only visually highlighted

### ISSUE THREAD

**Issue by Ram-G** [Azure/LogicAppsUX#3548]
### Describe the Bug with repro steps

1. Create a new flow in designer standalone (nx serve).
2. Add a "append to string variable" action (or any action that has required fields).
3. Open screen reader (nvda).
4. Tab over the 'required' fields. 
5. Observe that error message shows up visually, but is not read aloud by screen reader.

This bug is similar to https://github.com/Azure/LogicAppsUX/issues/3528. The difference being that bug only talks about 'required' information not being mentioned by screen reader, this bug specifically says that the error message should be read by screen reader.

### What type of Logic App Is this happening in?

Consumption (Portal)

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Screenshots or Videos

![image](https://github.com/Azure/LogicAppsUX/assets/10837129/a5fcf02b-57c2-4093-ab80-aa35b42cf87c)

### Browser

Edge

### Additional context

Related internal bug: https://msazure.visualstudio.com/One/_workitems/edit/13176020

AB#25622079
